core/sail/extensible-store/src/main/java/org/eclipse/rdf4j/sail/extensiblestore/ExtensibleStore.java

```diff
@@ -157,4 +171,11 @@ public EvaluationStatisticsEnum getEvaluationStatisticsType() {
 	public ExtensibleStatementHelper getExtensibleStatementHelper() {
 		return ExtensibleStatementHelper.getDefaultImpl();
 	}
+
+	public enum Cache {
```

Comment:
I am not aware of the full context of this change, but I find the cache terms suggested here a bit "unusual". What about [node, lazy, eager] instead? Or is the "dynamic" cache really dynamic?

Reply:
Thank! I've renamed them now.

Reply:
Do we have a naming standard for enums in this project? Looking at other enums defined in the project, it seems like all-caps is the most common. Personally, I prefer pascal case, so [None, Lazy, Eager] in this case. 😉

Reply:
We follow the Eclipse coding conventions (with a few minor adjustments, see https://github.com/eclipse/rdf4j/blob/main/CONTRIBUTING.md#code-formatting). Since enum instances are constants, they should follow the conventions for contants (which are ALL_CAPS).